.github/workflows/release.yml

```diff
@@ -69,6 +74,10 @@ jobs:
     if: startsWith(github.ref, 'refs/tags/v')
     runs-on: ubuntu-latest
     needs: build
+    permissions:
+      checks: write
+      contents: write
+      pages: write
```

Comment:
You probably don't need `pages: write` since it's not directly triggering a pages build.

I don't think it needs any other permissions in order to create the branch and push the updated docs, and I suspect that it does _not_ need `pull-requests: write` because you generally don't need any special permissions to create a new PR.

Reply:
I don't think you need `checks: write` for this one either.

I'm willing to try it with just `contents: write`, and see what we learn.